components/Layout/StyledHeadlineTwo/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import { croissantOne } from "../Header";
+
+export default styled.h2`
+  text-align: center;
+  color: var(--light-brown);
+  font-family: ${croissantOne.style.fontFamily};
```

Comment:
```suggestion
  font-family: ${croissantOne.style.fontFamily}, sans-serif;
```

**Additionally:**
In my opinion the name of the component is a bit confusing. I would suggest to call this component `StyledSublineLine`.